ansible_collections/ds/ansible_ds/tests/README.md

```diff
@@ -0,0 +1,16 @@
+389 Directory Server Ansible collection tests
+==========================
+
+This directory contains the test framework
+
+Note: tests are either:
+    - test_xxxx methods within python file or 
+    - test_xxxx.yml playbooks
+
+They can be run:
+    - directly from pytest. (Usefull when wanting to run a single test: i.e pytest $PWD -k test_gather_dsinst_info.yml)
+    - through ansible-test i(i.e: ansible-test units)
```

Comment:
It gives the next error on a clean VM install:

```
gw0 [1]
F                                                                                                                                                                                       [100%]
========================================================================================== FAILURES ===========================================================================================
_________________________________________________________________________________________ test_info1 __________________________________________________________________________________________
[gw0] linux -- Python 3.10.4 /usr/bin/python3
topo_m2 = <lib389.topologies.TopologyMain object at 0x7fe0d15df400>
ansibletest = <ansible_collections.ds.ansible_ds.tests.conftest.AnsibleTest object at 0x7fe0d15dc9a0>

    def test_info1(topo_m2, ansibletest):
        """Test dsinfo module #1.
            Setup: two suppliers
            Step 1: Run ds_info module
            Step 2: Verify The output
            Result 1: No error
            Result 2: check that supplier1 and supplier2 instances are in the facts.
                      check that supplier1 userroot backend is in the facts.
                      check that supplier2 userroot backend is in the facts.
        """
    
        log = ansibletest.getLog(__name__)
        args = { "prefix" : os.getenv('PREFIX', '') }
>       result = ansibletest.runTestModule( { "ANSIBLE_MODULE_ARGS": args } )

tests/unit/plugins/modules/test_ds_info.py:40: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/conftest.py:159: in runTestModule
    return self.runModule(cmd, stdin_text)
tests/conftest.py:152: in runModule
    self.result = json.loads(result.stdout)
/usr/lib64/python3.10/json/__init__.py:346: in loads
    return _default_decoder.decode(s)
/usr/lib64/python3.10/json/decoder.py:337: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <json.decoder.JSONDecoder object at 0x7fe0d2f1bbe0>
s = '[general]\nversion = 2\nstart = True\n\n[slapd]\ninstance_name = ansible-default\nroot_password = ZECnyKawbKqN5fbZI.m...pd_directory": "/var/lib/dirsrv/slapd-supplier1/db"}}, "prefix": ""}, "invocation": {"module_args": {"prefix": ""}}}\n'
idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` beginning with
        a JSON document) and return a 2-tuple of the Python
        representation and the index in ``s`` where the document ended.
    
        This can be used to decode a JSON document from a string that may
        have extraneous data at the end.
    
        """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration as err:
>           raise JSONDecodeError("Expecting value", s, err.value) from None
E           json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)
```